subseas_flex_fcst/maproom.py

```diff
@@ -308,8 +308,10 @@ def local_plots(marker_pos, start_date, lead_time):
     cdf_graph.update_traces(mode="lines", connectgaps=False)
     if CONFIG["time_units"] == "days":
         start_date_pretty = (pd.to_datetime(start_date)).strftime("%-d %b %Y")
-    else:
+    elif CONFIG["time_units"] == "months":
         start_date_pretty = (pd.to_datetime(start_date)).strftime("%b %Y")
+    else:
+        start_date_pretty = "some day"
```

Comment:
Is this a legitimate case? I thought we would raise an exception if the units are none of the expected ones.

Reply:
it's not a legitimate case but I didn't feel it was worth stopping the program from making the graphs just because we don't know what dates to print... Right now we are using these units for printing dates only. Maybe I have the illegitimate cases print an error title?

Reply:
My philosophy is that when something unexpected happens, it's better to fail loudly and as early as possible, rather than silently falling back to some default value. It's easier to fix a problem immediately, when you still remember what you did.